cmd/nerdctl/volume_prune_linux_test.go

```diff
@@ -26,19 +26,21 @@ import (
 func TestVolumePrune(t *testing.T) {
 	base := testutil.NewBase(t)
 	tID := testutil.Identifier(t)
-	base.Cmd("volume", "prune", "-f").Run()
+	base.Cmd("volume", "prune", "-a", "-f").Run()
 
+	vID := base.Cmd("volume", "create").Out()
 	base.Cmd("volume", "create", tID+"-1").AssertOK()
 	base.Cmd("volume", "create", tID+"-2").AssertOK()
 
 	base.Cmd("run", "-v", fmt.Sprintf("%s:/volume", tID+"-1"), "--name", tID, testutil.CommonImage).AssertOK()
 	defer base.Cmd("rm", "-f", tID).Run()
 
-	base.Cmd("volume", "prune", "-f").AssertOutContains(tID + "-2")
+	base.Cmd("volume", "prune", "-f").AssertOutContains(vID)
+	base.Cmd("volume", "prune", "-a", "-f").AssertOutContains(tID + "-2")
```

Comment:
@yankay @AkihiroSuda 

Curious is this a breaking change which changes the behaviour of `nerdctl volume prune -f` without `-a`? Or any breaking change which is for Docker compliance is not considered as breaking change in Nerdctl?

Docker bumped major version with this change so fine in Docker side. https://docs.docker.com/engine/release-notes/23.0/#bug-fixes-and-enhancements-1